src/Symfony/Component/HttpKernel/Event/GetResponseForExceptionEvent.php

```diff
@@ -64,4 +69,24 @@ public function setException(\Exception $exception)
     {
         $this->exception = $exception;
     }
+
+    /**
+     * Mark the event as allowing a custom, non-500 response code.
+     *
+     * @param bool $allowCustomResponseCode
+     */
+    public function setAllowCustomResponseCode($allowCustomResponseCode)
```

Comment:
I would named this `allowCustomResponseCode()` and the other one `isAllowingCustomResponseCode`. WDYT?